explore-the-admin-ui.md

```diff
@@ -4,4 +4,13 @@ toc: false
 feedback: false
 ---
 
-Docs coming soon. For guidance on accessing the Admin UI, see [Start a Local Cluster](start-a-local-cluster.html) and [Manual Deployment](manual-deployment.html).
\ No newline at end of file
+CockroachDB's Admin UI provides detail about your cluster's performance and health.
+
+You can access the CockroachDB Admin UI via HTTP from any node in a cluster using either:
+
+- The default port `8080`. For example, `http://<node IP address>:8080`.
```

Comment:
Think this might still be a little misleading. You can only use `8080` if you don't specify `--http-port` when starting one of the nodes. Also just realized that we may need to mention the `--http-host` flag as well. What about something like:

> When starting each node with [`cockroach start`](start-a-node.html), you use the `--http-host` and `--http-port` flags to specify the address and port for HTTP requests. You can access the CockroachDB Admin UI using any of these address/port combinations, for example, `http://<node IP address>:<custom port>`.

Thoughts? Probably too complicated now...